docs/docs/reference/configuration.md

```diff
@@ -866,7 +866,7 @@ errors:
 authorizers:
   ## Deny ##
   #
-  # The [`deny` authorizer](https://www.ory.sh/oathkeeper/docs/pipeline/authz#allow).
+  # The [`deny` authorizer](https://www.ory.sh/oathkeeper/docs/pipeline/authz#deny).
```

Comment:
This is autogenerated from here ( https://github.com/ory/oathkeeper/blob/master/.schema/config.schema.json ) and needs to be fixed there :)

Reply:
doh.  i'll close this and fix it in another PR to keep the history clean. :)